content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +27,75 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
    :guilabel:`Virtual Locations/Scrap` location.
 
-Scrap manufacturing components
-==============================
-
-To scrap components during the manufacturing process, begin by navigating to
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+.. _scrap_components/scrap-window:
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Open the Scrap pop-up window
+============================
 
-.. image:: scrap_manufacturing/scrap-button.png
-   :align: center
-   :alt: The scrap button on a manufacturing order.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window. Once
+opened, the pop-up window can be filled out in the same way, regardless of how it was accessed.
+However, the method for accessing the window differs depending on the type of scrapping taking
+place.
```

Comment:
Can you help me elaborate what you meant by the different kind of scrapping? Are you referring to the different subsections below?

In that case, how about rewording this sentence to something like:

```suggestion
The
following are three ways of navigating to the :guilabel:`Scrap` window, depending on the app and
product type.
```